### DISCUSSION TITLE
simple dialog customization

### DISCUSSION THREAD

**Discussion by pas059** [wixtoolset/issues#7660] [Questions] [answered]
Hi,
Is there a simple way to change the size of an installer (something like putting width="400" height="300" somewhere) in WiX 4?

Is it possible to replace a standard dialog (for instance FeaturesDlg) in WiX 4?

**Comment by robmen** [comment, marked as answer]
> Is there a simple way to change the size of an installer (something like putting width="400" height="300" somewhere) in WiX 4?

No. That is not a scenario that has been optimized for.

> Is it possible to replace a standard dialog (for instance FeaturesDlg) in WiX 4?

Yes. Same as in v3.
